Script/AtomicEditor/ui/frames/inspector/ColorChooser.ts

```diff
@@ -121,6 +124,16 @@ class ColorChooser extends Atomic.UIWindow {
 
         Atomic.ui.blockChangedEvents = true;
 
+        var colorhist = Preferences.getInstance().colorHistory;
+        (<Atomic.UIColorWidget>this.getWidget("chistory0")).setColorString(colorhist[0]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory1")).setColorString(colorhist[1]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory2")).setColorString(colorhist[2]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory3")).setColorString(colorhist[3]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory4")).setColorString(colorhist[4]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory5")).setColorString(colorhist[5]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory6")).setColorString(colorhist[6]);
+        (<Atomic.UIColorWidget>this.getWidget("chistory7")).setColorString(colorhist[7]);
+
```

Comment:
If we want more history buttons I'll change it.  8 worked out because of the current button sizing, but from an artist's perspective, 8 may not be enough.